algorithms/kernel/kmeans/kmeans_csr_lloyd_distr_step2_fpt_dispatcher.cpp

```diff
@@ -28,6 +28,32 @@ namespace daal
 {
 namespace algorithms
 {
-__DAAL_INSTANTIATE_DISPATCH_CONTAINER(kmeans::DistributedContainer, distributed, step2Master, DAAL_FPTYPE, kmeans::lloydCSR)
+__DAAL_INSTANTIATE_DISPATCH_CONTAINER(kmeans::interface2::DistributedContainer, distributed, step2Master, DAAL_FPTYPE, kmeans::lloydCSR);
```

Comment:
no need to explicitly specify kmeans::interface2, using should be added in header file for interface2.